src/components/Header/index.jsx

```diff
@@ -2,12 +2,13 @@ import React from 'react'
 import Logo from '../../assets/logo.png'
 import './styles.scss'
 
-export default function Header () {
+// eslint-disable-next-line react/prop-types
+export default function Header ({ onHandleLinkPage }) {
   return (
     <section>
       <header className='header'>
         <section className="header--logo">
-          <img src={Logo} alt="Logo" />
+          <img src={Logo} alt="Logo" onClick={() => onHandleLinkPage()} />
```

Comment:
`onClick` expects a function, in this case an anonymous function is created to call `onHandleLinkPage` with no params, here you can just pass `onHandleLinkPage`
```suggestion
          <img src={Logo} alt="Logo" onClick={onHandleLinkPage} />
```